server/sonar-server/src/main/java/org/sonar/server/ws/WebServiceEngine.java

```diff
@@ -120,7 +120,9 @@ public void execute(Request request, Response response) {
         return;
       }
       LOGGER.error("Fail to process request " + request, e);
-      sendErrors(response, 500, new Errors().add(Message.of(e.getMessage())));
+      // Sending exception message into response is a vulnerability. Error must be
+      // displayed only in logs.
+      sendErrors(response, 500, new Errors().add(Message.of("error_occurred")));
```

Comment:
ok, that's "bourrin" and will affect all errors and not only ES ones, but that's good